src/scripts/productsTabs.js

```diff
@@ -0,0 +1,60 @@
+const furnitureCategoryTabs = document.querySelectorAll('.category-tab-select');
+
+furnitureCategoryTabs.forEach(function (tab) {
+  tab.addEventListener('click', checkResolutionOfScreen);
+});
+
+function checkResolutionOfScreen (event) {
+  const targetTab = event.currentTarget.getAttribute('href');
+  const tabPanelId = targetTab.substring(1, targetTab.length);
+  const tabPanel = document.getElementById(tabPanelId);
```

Comment:
jesli uzylbys tu `querySelector` to nie musiałbyś wycinać tego `#` w `tabPanelId`

Reply:
Zamieniłem na `querySelector`